package_test.go

```diff
@@ -29,6 +30,24 @@ func WithoutArgs() {
 	})
 }
 
+func WithKeyPairValues() {
+	Convey("Given summon-aws-secrets is retrieving a secret with key value format", func() {
+		secret := "{\"username\":\"USERNAME\",\"password\":\"PASSWORD\",\"port\":8000}"
```

Comment:
I think back-ticks would be useful here for better readability without all the escapes:
```
secret := `{ "username": "USERNAME", "password": "PASSWORD", "port": 8000 }`
```